src/draw.c

```diff
@@ -144,6 +157,35 @@ static void createPlayer(void)
     }
 }
 
+static void createEnemy(int x, int y)
```

Comment:
pretty much the same as create player - why not pass a pointer to a ship and create it?